src/fragments/lib/in-app-messaging/js/clear-messages.mdx

```diff
@@ -0,0 +1,12 @@
+Once messages have been synced to your user’s device, you may find that you want to clear them out for any reason. `InAppMessaging.clearMessages()` can be used to clear the synced messages from the user’s device.
```

Comment:
These two sentences seem repetitive. IMO just the second one conveys the message clearly. 
`InAppMessaging.clearMessages()` can be used to clear the synced messages from the user’s device.

Reply:
I have merged it into a single statement.